components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminService.java

```diff
@@ -571,20 +569,20 @@ public boolean isScopeExist(String scope) throws IdentityOAuthAdminException {
      */
     public void updateConsumerAppState(String consumerKey, String newState) throws IdentityOAuthAdminException {
 
-        OAuthAppDAO oAuthAppDAO = new OAuthAppDAO();
         try {
             OAuthAppDO oAuthAppDO = AppInfoCache.getInstance().getValueFromCache(consumerKey);
             if (oAuthAppDO == null) {
-                oAuthAppDO = oAuthAppDAO.getAppInformation(consumerKey);
+                oAuthAppDO = getOAuthApp(consumerKey);
```

Comment:
We can directly use 'getOAuthApp' method being newly implemented right, rather than retrieving from the cache specifically first

Reply:
Yes :)

Reply:
Updated the code to use the getOAuthApp method